src/components/_classes/component/Component.js

```diff
@@ -2249,7 +2249,7 @@ export default class Component extends Element {
       .then((editor) => {
         editor = editor.edit(element);
         editor.removeAllListeners('change');
-        editor.setOptions(settings);
+        editor.setOptions(_.omit(settings, 'isUseWorkerDisabled'));
```

Comment:
I would rather use ```_.pick``` so we won't have to keep introducing new "omits" as we add other settings elsewhere.